experimental/cloudstrapper/playbooks/roles/aws-cleanup/tasks/orc8r-secgroup.yaml

```diff
@@ -17,7 +16,36 @@
     factVpcId: "{{ valVpc.vpcs[0].vpc_id }}"
   when: valVpc.vpcs is defined and valVpc.vpcs[0].vpc_id is defined
 
-- name: gather all security groups for this vpc
+- name: locate eks_cluster_sg group
```

Comment:
In case you wanted to use existing pattern, I had done the following 
```
- name: Change all security groups in this VPC
  ec2_group:
    name: "{{ item.group_name }}"
    description: "{{ item.description }}"
    group_id: "{{ item.group_id }}"
    purge_rules: True
  when: vpc_info and vpc_info.vpcs | length > 0 and item.group_name != "default"
  with_items:
    - "{{ sec_info.security_groups }}"
  ignore_errors: true
  tags: cleanup

- name: Delete all security groups in this VPC
  ec2_group:
    group_id: "{{ item.group_id }}"
    state: absent
  when: vpc_info and vpc_info.vpcs | length > 0 and item.group_name != "default"
  with_items:
    - "{{ sec_info.security_groups }}"
  ignore_errors: true
  tags: cleanup
```